app/api/v2/views/user_views.py

```diff
@@ -35,26 +34,16 @@ class MyAdmin(Resource, UsersModel):
     def __init__(self):
         self.db = UsersModel()
 
-    # @jwt_required
-    # def delete(self, id):
-
-    #     delete_user = self.db.delete_user(id)
-
-    #     if delete_user:
-    #         return make_response(jsonify({
-    #             'status': 200,
-    #             "data": [{
-    #                 "id": id,
-    #                 "message": "User with that record has been deleted"
-    #             }]
-    #         }))
-    #     return make_response(jsonify({
-    #         'status': 200,
-    #         "data": [{
-    #             "id": id,
-    #             'message': 'User not found'
-    #         }]
-    #     }), 404)
+    @jwt_required
+    def delete(self, id):
+        """ Docstring for deleting users"""
+        app_users = self.db.get_user_id(id)
+        if not app_users:
+            return {"error": "User with that id not found"}, 404
+        delete_user = self.db.delete_user(id)
+
+        if delete_user:
+            return {"data": {"id": id, "message": delete_user}}, 200
```

Comment:
return only one dictionary

Reply:
Thank you for the feedback, I have resolved this.